### DISCUSSION TITLE
Sonoff SNZB-02 MQTT o eWeLink?

### DISCUSSION THREAD

**Discussion by ghost** [indomus/forum#1398] [HUB: Home Assistant]
Ho aggiunto un Sonoff SNZB-02 tramite Zigbee2MQTT e funziona correttamente visto che vedo tutti i dati e l'ho collegato al broker MQTT.
Ora, volevo sapere quali sono i vantaggi/svantaggi di usare MQTT o la repo di eWeLink con la sua dashboard.

**Comment by indomus** [comment]
[leggi qui](https://indomus.it/formazione/integrazioni-local-cloud-e-stati-via-polling-push-come-la-domotica-dialoga-coi-componenti/)

**Reply by ghost** [reply]
Conosco la differenza "basica" di un sistema locale ed uno cloud, però chiedevo a chi magari avesse installato o usasse eWeLink se ci sono degli effettivi vantaggi o cose "in più" che non si possono fare con ha o features in più :)

**Reply by snakuzzo** [reply]
Innanzi tutto non hai collegato il device al broker MQTT. Tu hai collegato il device al coordinator che viene gestito dal  coordinator tramite  zigbee2mqtt che a sua volta ti permette di gestire il tutto tramite un broker MQTT.

Ad ogni modo dipende da quello che ti serve.
Non è che in generale c'è "IL" modo migliore di fare una cosa.
Poi per quanto riguarda nello specifico il protocollo Zigbee sei comunque vincolato a quello che prevede il device, non è che tu possa fare qualcosa di così diverso. In questo caso il vantaggio principale è legato all'utilizzo di un coordinator unico che utilizzeresti eventualmente anche per altri device di altri produttori/marche
